### ISSUE TITLE
Allow to group colors with a tag

### ISSUE THREAD

**Issue by WimeSTone** [aseprite/aseprite#1575]
I got a lot of small 4-5 color palettes for different purposes, but I wish I had them all in one ~128 sized palette. Eventually, it becomes messy and sometimes ends up in inserting some unused colors so that palette align is one sub-palette per row. The request is the following:

- allow to group colors under a name, colors don't lose their original indices;
- allow to add an already existing palette or a part of it as a group;
- allow to reorganize color groups layout without colors losing their original indices;
- colors in color group preserve all original editing/moving functions.

Optional:
- allow a color to be a part of different groups, modifying such color will prompt user (can be disabled), where they can choose to either modify original color in all groups or add a new one to the master palette.

Something like this:
![palettegroups](https://user-images.githubusercontent.com/12399059/30633037-e5dc8cb4-9df2-11e7-8b90-aba965d39c55.png)
It is always possible to switch back to normal palette view.

Pros:
- you keep your colors organized;
- no more reference sprites;
- other artists in your team don't have a need to recheck colors.

Cons:
- where to store this? In *.ase directly or separately as a new format? Document template?

**Comment by WimeSTone** [comment]
Moved to community site, my apologies.

**Comment by Tranberry** [comment]
What community site is that?

**Comment by WimeSTone** [comment]
@Tranberry https://blog.aseprite.org/2017/06/20/aseprite-community/
tl;dr Git is now for technical issues, community site - for requests and posting art.

**Comment by dacap** [comment]
Hi there! Actually you can use any of both sites. Thanks for this feature requests, I'll take a look in the community site.

**Comment by dacap** [comment]
I'm opening this issue to continue the development discussion from [here](https://community.aseprite.org/t/step-one-dont-break-it/609).

**Comment by pazon12** [comment]
Yup. But now that I look at this, switching palette views might also be a bit of a task. Which makes me want to store the palette grouping on the _palette_ than the sprite file.

**Comment by dacap** [comment]
@pazon12 maybe a sprite could have a link to another `.aseprite` file that contains the palette. So palettes can be stored in any file and files can link other palettes. (anyway this is close to the [projects feature](https://github.com/aseprite/aseprite/issues/126))

**Comment by pazon12** [comment]
Oh. Projects. I was even thinking of that. That might even work better. You really  would group palettes for a project. versus a single sprite, as most people would use the same palette for multiple things. I don't know its just throwing that out there. .asep might become a thing.

**Comment by cyraid** [comment]
Sorry I realize now you said *development* discussion after I posted a message in the community link you stated above, sorry about that!
